TL improvements (#346)
07th-mod/higurashi-console-arcs: Update/omot_013.txt

//r――普段着で来るように、とレナからそれとなく伝えてくれと頼んでおいたのだけど、どうやらちゃんと守ってくれたようだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "——普段着で来るように、とレナからそれとなく伝えてくれと頼んでおいたのだけど、どうやらちゃんと守ってくれたようだ。",

> 			NULL, "...I asked Rena to suggest indirectly her coming dressed in everyday clothes, and she seemed to have accepted Rena's suggestion.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...I asked Rena to indirectly suggest she should come dressed in everyday clothes, so she must have listened to Rena.", Line_Normal);
```

	ClearMessage();

//rまぁ、魅音のおしゃれ姿もちょっと見てみたかったが…変に気合を入れられると余計にぎくしゃくしそうな気がしたから、それは次回の楽しみということで。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まぁ、魅音のおしゃれ姿もちょっと見てみたかったが…変に気合を入れられると余計にぎくしゃくしそうな気がしたから、それは次回の楽しみということで。",

> 			NULL, "Well, I did kinda want to see Mion dressed up in a fashionable outfit... but I felt if she get too fired up, she'll be more awkward, so I saved the fun for the nest time.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Well, I did kinda want to see Mion dressed up in a fashionable outfit... but I felt she'd get more awkward if she got too fired up, so I saved the fun for next time.", Line_Normal);
```

//r――普段着で来るように、とレナからそれとなく伝えてくれと頼んでおいたのだけど、どうやらちゃんと守ってくれたようだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "——普段着で来るように、とレナからそれとなく伝えてくれと頼んでおいたのだけど、どうやらちゃんと守ってくれたようだ。",

> 			NULL, "...I asked Rena to suggest indirectly her coming dressed in everyday clothes, and she must have done as Rena's suggestion.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...I asked Rena to indirectly suggest she should come dressed in everyday clothes, so she must have listened to Rena.", Line_Normal);
```

//rおかげで、『一体俺たちは何をしに行くんだよっ？』などと基本中の基本を確認せざるを得ないくらいに、完全パニック状態だったもんな。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかげで、『一体俺たちは何をしに行くんだよっ？』などと基本中の基本を確認せざるを得ないくらいに、完全パニック状態だったもんな。",

> 			NULL, "She was in a complete panic in that way, so I couldn't help asking her \"What on earth are we going to do?\" to determine the most basic of basics in our situation.", Line_Normal);

Reviewer: ```suggestion
			NULL, "She was in a complete panic, so I couldn't help asking her \"What on earth are we going to do?\" to determine the most basic of basics in our situation.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101189", 256, TRUE);
	OutputLine(NULL, "周りに誰がいてもいなくても、それだけは同じだろ？」",

> 			NULL, " That's the same, even if there's anybody else around us or not, right?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " That's the same whether there's anybody else around us or not, right?\"", Line_Normal);
```

			NULL, "\"I've been thinking waaay too hard~.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300577", 256, TRUE);
	OutputLine(NULL, "私ってほら、話をする男子って年下か、オヤジばっかりだったでしょ？",

> 			NULL, " Male I talk to was only a boy younger than I or middle-aged man, right?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " All the guys I ever talk to are either younger boys or middle-aged men, right?", Line_WaitForInput);
```

			NULL, "\"No, well... Yeah, I suppose that's true...", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300580", 256, TRUE);
	OutputLine(NULL, "なんていうか、意識しちゃったらどうも、レナたちとは調子が違ってきてさ～」",

> 			NULL, " I mean, it seems that I'm conscious of you so I have the different attitude to you from Rena and the others.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I mean, once I became conscious of it I started to have a different attitude compared to Rena and the others.\"", Line_Normal);
```

//rどれくらい初歩かって？　…いや、仮にも園崎家ご令嬢の名誉のために、おそらく受験勉強をやってきたやつなら『激怒する』レベルとだけ言っておこう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "どれくらい初歩かって？　…いや、仮にも園崎家ご令嬢の名誉のために、おそらく受験勉強をやってきたやつなら『激怒する』レベルとだけ言っておこう。",

> 			NULL, "Just how far back is she? ...No, for the honor of the Sonozaki family's heiress, I evasively answer that whoever studied for entrance exam would probably be <i>enraged</i> to hear that she was only at this level in her studies.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Just how far back is she? ...No, for the honor of the Sonozaki family's heiress, anyone who studied for entrance exam would probably be <i>enraged</i> to hear that she was only at this level in her studies.", Line_Normal);
```

//rもしやと思った俺は、魅音のノートを取り上げて他の部分をチェックする。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "もしやと思った俺は、魅音のノートを取り上げて他の部分をチェックする。",

> 			NULL, "I decided to take away Mion's notes to see how she answered the other parts.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I decided to grab Mion's notes to see how she answered the other parts.", Line_Normal);
```

			NULL, "\"Besides~!", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300597", 256, TRUE);
	OutputLine(NULL, "　うちの学校は世界の国の首都が言えたら、合格できるレベルなんだよ？",

> 			NULL, " Our choice school is at a level that I pass just if I know the capitals of other countries of the world, you know?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Our school is at a level where you can pass if you just know the capitals of other countries of the world, you know?", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101220", 256, TRUE);
	OutputLine(NULL, "「いくら運命が人の意志の結晶と梨花ちゃんが言っても、今の学力で受験を乗り切れるのなら、人はウナギで空を飛べるわっっ！！」",

> 			NULL, "\"No matter how much Rika-chan says that fate is the crystallization of human will, if you can survive the entrance exams at your current ability, then people can fly through the sky riding on a eel!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"No matter how much Rika-chan says that fate is the crystallization of human will, if you can survive the entrance exams at your current ability, then people can fly through the sky riding on eels!!\"", Line_Normal);
```

			NULL, "\"I...", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300614", 256, TRUE);
	OutputLine(NULL, "一緒の学校に、行きたいなと思って！　…ね」",

> 			NULL, " I was thinking, it would be nice to go to highschool together! ...Okay?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I was thinking, it would be nice to go to school together! ...Okay?\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101232", 256, TRUE);
	OutputLine(NULL, "「い、いいいい、一緒って言っても、お前俺より一学年上だろ？！」",

> 			NULL, "\"E-e-e-even if you say you want us to go to highschool together, you're a year older than me, aren't you?!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"E-e-e-even if you say you want us to go to school together, you're a year older than me, aren't you?!\"", Line_Normal);
```

	//r思わず裏返ってしまった声で、俺は懸命に言葉をつなぐ。…いや、だから落ち着けっての。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "思わず裏返ってしまった声で、俺は懸命に言葉をつなぐ。…いや、だから落ち着けっての。",
				NULL, "My voice had suddenly cracked and I was desperately trying to string together the words. ...Come on, I need to stay calm.", Line_Normal);
		ClearMessage();

	//rにしても、…別に同じ学校目指すからって、そのためにわざわざ受験勉強を頑張る必要なんて…。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "にしても、…別に同じ学校目指すからって、そのためにわざわざ受験勉強を頑張る必要なんて…。",

> 				NULL, "Anyway... even if her goal is to get into the same school, I wonder she's going to have to work really hard studying for exams...", Line_Normal);

Reviewer: ```suggestion
				NULL, "Anyway... even if her goal is to get into the same school, that doesn't mean she has to work really hard studying for exams...", Line_Normal);
```

			NULL, "\"But, well, I thought it over a bit.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300632", 256, TRUE);
	OutputLine(NULL, "私たちの村も、どんどん変わってきている」",

> 			NULL, " Our village is rapidly changing.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Our village is steadily changing.\"", Line_Normal);
```

//r俺はそこで一度言葉を切って、思い出したように言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "俺はそこで一度言葉を切って、思い出したように言った。",

> 			NULL, "I once stopped talking, then continued like hitting on what to say.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I cut myself off there, then continued like I remembered something.", Line_Normal);
```

			NULL, "\"Hee hee hee...", Line_WaitForInput);
	ModPlayVoiceLS(4, 9, "ps3/s14/09/210900027", 256, TRUE);
	OutputLine(NULL, "今日はデートかしら？」",

> 			NULL, " Are you on a date today?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Are you on a date?\"", Line_Normal);
```

//rデートと言われて、魅音はアセアセと落ち着かなくなる。…どうやら、俺のギクシャクした動きは、緊張だと思われたらしい。…よかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "デートと言われて、魅音はアセアセと落ち着かなくなる。…どうやら、俺のギクシャクした動きは、緊張だと思われたらしい。…よかった。",

> 			NULL, "Hearing the word 'date', Mion got restless. ...And Takano seemed to think the reason for my jerky movements was just that I was nervous. ...Great.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Hearing the word \"date\", Mion got restless. ...And Takano seemed to think the reason for my jerky movements was just that I was nervous. ...Great.", Line_Normal);
```

	ClearMessage();

	ModDrawCharacter(5, 6, "sprite/si1a_akuwarai_a1_", "0", -160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 18, 200, TRUE );

//rそんな様子をからかうように見つめて、詩音は俺にニヤリと笑ってきた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そんな様子をからかうように見つめて、詩音は俺にニヤリと笑ってきた。",

> 			NULL, "When Shion saw me like that with a grin, she looked like she was getting ready to make fun of me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "When Shion saw me like that, she grinned and looked like she was getting ready to make fun of me.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101259", 256, TRUE);
	OutputLine(NULL, "「いや、まぁ…興味っていうか、それ…」",

> 			NULL, "\"Oh, well... I guess you could say I'm interested, I mean those...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Oh, well... I guess you could say I'm interested...\"", Line_Normal);
```

//r…そうか、今日は二人きりの勉強会。…いや、勉強会という『デート』だったんだ。それで無視されたら、魅音でなくても怒るだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そうか、今日は二人きりの勉強会。…いや、勉強会という『デート』だったんだ。それで無視されたら、魅音でなくても怒るだろう。",

> 			NULL, "...Right, we're studying today, just the two of us. ...No, it's a studying \"date.\" So anyone who is ignored in that situation will get upset.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Right, we're studying today, just the two of us. ...No, it's a studying \"date.\" So anyone ignored in that situation would get upset.", Line_Normal);
```

//rその問いつめる勢いに押されたのか、急に魅音はしどろもどろになる。そんな様子が余計に詩音の逆鱗に触れたようで、詩音は怒りもあらわに魅音に詰め寄った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その問いつめる勢いに押されたのか、急に魅音はしどろもどろになる。そんな様子が余計に詩音の逆鱗に触れたようで、詩音は怒りもあらわに魅音に詰め寄った。",

> 			NULL, "The force of that question seemed to suddenly make Mion falter. Shion's overwhelming anger had turned the situation around. Now she was the one venting her rage at Mion, getting her face closer to Mion's.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The force of that question seemed to suddenly make Mion falter. Shion's overwhelming anger had turned the situation around. Now she was the one venting her rage at Mion, moving her face in closer.", Line_Normal);
```

//r…俺にはそれが、嫌な話を聞かせまいとする思いやりからの逡巡だとわかっていたので、逆に安心させるように笑って言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…俺にはそれが、嫌な話を聞かせまいとする思いやりからの逡巡だとわかっていたので、逆に安心させるように笑って言った。",

> 			NULL, "...I realized that she was trying to avoid telling these bad stories out of concern for me, that's why I smiled and tried to reassure her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...I realized that she was trying to avoid telling these bad stories out of concern for me. That's why I smiled and tried to reassure her.", Line_Normal);
```

//r確かに詩音の反応は過剰だったかもしれないが、…落ち着いて考えると、その根底にある気持ちをうかがい知ることが出来るような…そんな、気がする。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "確かに詩音の反応は過剰だったかもしれないが、…落ち着いて考えると、その根底にある気持ちをうかがい知ることが出来るような…そんな、気がする。",

> 			NULL, "It's true that Shion's reaction was a bit excessive... but judging from it calmly, I learned what feelings drove her to do that... maybe.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It's true that Shion's reaction was a bit excessive... but thinking about it calmly, I realized what underlying feelings drove her... I think.", Line_Normal);
```
